doc/ssl/SSL_SESSION_get_time.pod

```diff
@@ -52,6 +59,8 @@ valid values.
 
 SSL_SESSION_set_time() and SSL_SESSION_set_timeout() return 1 on success.
 
+SSL_SESSION_get0_hostname returns the current value.
```

Comment:
Right - so you need to say something about under what conditions a NULL is returned (presumably if no SNI has been sent).